### ISSUE TITLE
What record package for flutter should I use?

### ISSUE THREAD

**Issue by AllenDang** [xuegao-tzx/whisper_flutter_new#3]
I tried https://pub.dev/packages/record, but always get a 44 kHz wav no matter what sample rate I set. 
What record package could generate 16 kHz wav audio?

**Comment by xuegao-tzx** [comment]
I use the ffmpeg library to achieve the conversion. You can see the sample code for details.

**Comment by AllenDang** [comment]
got it, thanks, I get it works. but the transcription speed is kind of slow, using small model for a 3 sec audio will takes 18 secs to finish.

**Comment by xuegao-tzx** [comment]
yes, I also found this problem, but there is no good solution at present. You can try to implement this new method to see the speed. You can try to use [this android app](https://play.google.com/store/apps/details?id=ink.xcl.aichats) or [this ios app](https://testflight.apple.com/join/GvRwZJAx). After registering and logging in, send a voice message through this app. When there is no network, the local model voice-to-text will be used. When there is a network, the API will be called for voice-to-text. You can take a look at the online and offline conversion time.

**Comment by xmanatee** [comment]
See https://github.com/xuegao-tzx/whisper_flutter_new/issues/4#issuecomment-2376677989
